build.gradle

```diff
@@ -3,9 +3,10 @@ plugins {
 }
 
 static def calcTags(version) {
+    def xenitVersion="1.0.1-SNAPSHOT"
     def tags = [
-            "${version.major}.${version.minor}.${version.rev}".toString(),
-            "${version.major}.${version.minor}".toString()
+        "${version.major}.${version.minor}.${version.rev}x${xenitVersion}".toString(),
```

Comment:
Can we also keep publishing the previous version numbers?
They would be "evergreen" versions that are regularly updated.

Reply:
I think we should just deprecate those versions in time, for clarity.
Otherwise people will just use those, nothing would change. And why would we even bother to release?